tests/test_end_to_end.py

```diff
@@ -172,3 +173,31 @@ def test_reference_grib(
     # various inconsistencies (of dtype casting int to float, etc.). With the right combination of
     # options passed to the pipeline, seems like these should pass?
     # xr.testing.assert_equal(ds.load(), ds2)
+
+
+@pytest.mark.parametrize("consolidate_dimension_coordinates", [False, True])
+def test_xarray_zarr_consolidate_dimension_coordinates(
+    netcdf_local_file_pattern_sequential,
+    pipeline,
+    tmp_target_url,
+    consolidate_dimension_coordinates,
+):
+    pattern = netcdf_local_file_pattern_sequential
+    with pipeline as p:
+        (
+            p
+            | beam.Create(pattern.items())
+            | OpenWithXarray(file_type=pattern.file_type)
+            | StoreToZarr(
+                target_root=tmp_target_url,
```

Comment:
do you mind seeing if this still works if we use `target_root=tmp_target` b/c that actually returns a `FSSpecTarget`?

Reply:
Great call! I'll give it a test.

Reply:
Gotta figure out a way to append the path to an `FSSpecTarget`. 
This fails: `store = zarr.open(os.path.join(tmp_target + "subpath"))`

Reply:
haha, if it doesn't just work then I'll deal with it in my PR so don't worry 😆